cypress.json

```diff
@@ -10,5 +10,6 @@
     }
   },
   "defaultCommandTimeout": 30000,
-  "requestTimeout": 30000
+  "requestTimeout": 30000,
+  "videoCompression": false
```

Comment:
Why no compression?

Reply:
Good call. I was under the assumption that Cypress was going to record and compress a video for every test which is not actually the case.